packages/azure-container-services/images/instana-azure-container-services/.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
```

Comment:
Question: Why do we need this file? 🤔 I thought the Dockerfiles only copy specific files into the container.